sunpy/map/mapbase.py

```diff
@@ -2052,7 +2052,8 @@ def peek(self, draw_limb=False, draw_grid=False,
                 pad = 0.12  # Pad to compensate for ticks and axes labels
             else:
                 pad = 0.05  # Default value for vertical colorbar
-            figure.colorbar(im, pad=pad)
+            figure.colorbar(im, pad=pad).set_label(str(self.unit)
```

Comment:
You should be able to pass `label=...` directly to the `colorbar()` call here.

Reply:
Yeah but the positioning of the label requires the set_label attribute right?

Reply:
The docs say "label='" is a keyword arg to colorbar.

Reply:
Yeah position of the label does require the ```set_label``` attribute